test/suite/test_checkpoint_snapshot05.py

```diff
@@ -147,11 +147,11 @@ def test_checkpoint_snapshot(self):
         try:
             ckpt.start()
             
-            # Wait for checkpoint to start before committing.
+            # Wait for checkpoint to start and acquire its snapshot before committing.
             ckpt_started = 0
             while not ckpt_started:
                 stat_cursor = self.session.open_cursor('statistics:', None, None)
-                ckpt_started = stat_cursor[stat.conn.txn_checkpoint_running][2]
+                ckpt_started = stat_cursor[stat.conn.txn_checkpoint_snapshot_acquired][2]
                 stat_cursor.close()
                 time.sleep(1)
```

Comment:
We no longer need to sleep after the stat is retrieved. Move this sleep to the place before the `stat_cursor` is opened.

Reply:
@kommiharibabu  Do you mean to move it before we open the `stat_cursor` inside the loop?

Reply:
Yes.

Reply:
Fixed in [07a0be7](https://github.com/wiredtiger/wiredtiger/pull/9063/commits/07a0be746803804f00856fb19b57680bba824969)